coderedcms/project_template/basic/project_name/settings/base.py

```diff
@@ -203,3 +203,7 @@
 # Tags
 
 TAGGIT_CASE_INSENSITIVE = True
+
+# Sets default for primary key IDs - change to BigAutoField if you prefer a 64-bit integar instead
+# See https://docs.djangoproject.com/en/3.2/ref/models/fields/#autofield
+DEFAULT_AUTO_FIELD = 'django.db.models.AutoField'
```

Comment:
It looks like new projects generated with Django 3.2 are using `BigAutoField`, so it would make sense for us to follow that, given that these templates are only used for brand new coderedcms projects.

https://docs.djangoproject.com/en/3.2/releases/3.2/#customizing-type-of-auto-created-primary-keys

Reply:
updated in next commit